tests/build-job-matrix.py

```diff
@@ -143,6 +143,16 @@ def update_config(parent: Dict[str, Any], child: Dict[str, Any]) -> Dict[str, An
 
         pytest_monitor_enabling_paths.append(path)
 
+    max_index_characters = len(str(args.duplicates))
```

Comment:
technically, I think this should be `args.duplicates-`. consider the case of `--duplicates 10`. you'll have indices 0-9